src/View/Admin/WooTab.php

```diff
@@ -474,6 +481,17 @@ private function get_store_information_settings() {
 					'true'  => esc_html__( 'Yes - check this box by default', 'cc-woo' ),
 				],
 			],
+			[
+				'title'   => esc_html__( 'Checkbox Filter Location', 'cc-woo' ),
+				'desc'    => esc_html__( 'Change filter location where checkbox is rendered.', 'cc-woo' ),
+				'type'    => 'select',
+				'id'      => self::CHECKBOX_LOCATION,
+				'default' => 'false',
+				'options' => [
+					'woocommerce_after_checkout_billing_form' => esc_html__( 'woocommerce_after_checkout_billing_form', 'cc-woo' ),
+					'woocommerce_review_order_before_submit'  => esc_html__( 'woocommerce_review_order_before_submit', 'cc-woo' ),
```

Comment:
I feel like people are going to be a little confused by a dropdown that says "woocommerce_after_checkout_billing_form" and "woocommerce_review_order_before_submit" and be like "what?". Instead of listing the hooks themselves, let's use a human friendly value for the text shown. "After Checkout Billing Form" and "Before order submit button" or similar.

Definitely a good start overall though, but I'm curious if there are other hooks in this area that could be used as well.

Reply:
Okay. @Oceas change
```
'options' => [
	'woocommerce_after_checkout_billing_form' => esc_html__( 'woocommerce_after_checkout_billing_form', 'cc-woo' ),
	'woocommerce_review_order_before_submit'  => esc_html__( 'woocommerce_review_order_before_submit', 'cc-woo' ),
],
```
...to...
```
'options' => [
	'woocommerce_after_checkout_billing_form' => esc_html__( 'After Checkout Billing Form', 'cc-woo' ),
	'woocommerce_review_order_before_submit'  => esc_html__( 'Before order submit button', 'cc-woo' ),
],
```

Reply:
Changed.